env/export.go

```diff
@@ -143,5 +143,10 @@ func unquoteShell(value string) string {
 	// quotes, for example, "this is a \" quote string"
 	value = strings.Replace(value, `\"`, `"`, -1)
 
+	// Replace a literal escaped backtick (\`) with just a backtick (`)
+	// Some variables (like passwords) can contain backticks that should be interpreted as literal backticks, rather than
+	// as the beginning of a subcommand
+	value = strings.Replace(value, "\\`", "`", -1)
```

Comment:
Hah, I was going to suggest you use Go's backtick strings so you don't need to escape the backslash, and then saw the next character in the string 🙃

Reply:
every time i wrote an escaped-escaped-backtick in this PR, i thought "man, this would be so much easier with a backtick string" and then realised ahaha